city_scrapers/spiders/il_governors_state_university.py

```diff
@@ -0,0 +1,201 @@
+import re
+from datetime import datetime
+
+from city_scrapers_core.constants import BOARD, COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class IlGovernorsStateUniversitySpider(CityScrapersSpider):
+    name = "il_governors_state_university"
+    agency = "Governors State University"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.govst.edu/BOT-Meetings/"]
+    base_url = "https://www.govst.edu"
+    time_re = r"(?i)([01]?\d)(:?\d*)\s*([ap]\.?m\.?)"
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+        """
+        for year_section in response.xpath('//div[@class="toggle-list"]/ul/li'):
+            year_elt = year_section.xpath('div[@class="title"]/h3/text()')
+            # sometimes the year is not present in the table dates, so grab it from the
+            # section heading as backup
+            year = year_elt.get().replace("Meeting Dates for ", "").strip()
+            for row in year_section.xpath('div[@class="content"]/table/tbody/tr'):
+                item = row.xpath("td")
+                title = self._parse_title(item)
+                if title is None:
+                    continue
+                meeting = Meeting(
+                    title=title,
+                    description=self._parse_description(item),
+                    classification=self._parse_classification(title),
+                    start=self._parse_start(item, year),
+                    end=self._parse_end(item),
+                    all_day=self._parse_all_day(item),
+                    time_notes=self._parse_time_notes(item),
+                    location=self._parse_location(item),
+                    links=self._parse_links(item),
+                    source=self._parse_source(response),
+                )
+
+                # if postponed or canceled appears in any of these columns, it means the
+                # meeting is canceled, so just pass in all the row text to _get_status
+                row_text = " ".join(row.css("* ::text").getall())
+                meeting["status"] = self._get_status(meeting, text=row_text)
+                meeting["id"] = self._get_id(meeting)
+
+                yield meeting
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title. The inner html of the first column varies
+        quite a bit - brs, divs, b tags - so figuring out what is the title based on
+        line position. Sometimes the "title" is only a date, so if all else fails,
```

Comment:
Thanks for this comment! It really helps clarify things, especially since there are so many edge cases here. It seems like when the title is only a date it's a full board meeting, so we could return "Board of Trustees" in those cases.

And this is a small thing that we haven't documented well, but we're trying to remove "Meeting" from the ends of titles if it isn't needed so we can drop it from any title except for "Special Meeting"